RELEASING.md

```diff
@@ -0,0 +1,241 @@
+the minor or the major versions bumped, but not both, when compared to the latest released
+version. Which one it will be depends on the recent changes that were added after the
+latest release was made. More on why this should always be true is in the [Release
+Process](#release-process) section. For example, if the latest version of
+`cardano-ledger-core` that was released to CHaPs is `1.20.1.0`, then there will be a
+section like this:
+
+```
+# Version history for `cardano-ledger-core`
+
+## 1.20.2.0
+
+* Add ...
+...
+```
+
+It is quite common to experience conflicts in the changelog, since that will be the most
+common section of the codebase being updated at the same time. With respect on conflicts
+resolution on the package version it is pretty easy, the highest value should usually win.
+
+With respect to the actual change log entries, there is no enforced order in which entries
+should be added. To the top, to the bottom or in the middle of the list, it is
+irrelevant. In fact, randomized insertion reduces conflicts, so feel free to add entries
+anywhere in the section. Most important is that we can always infer the actual order from
+the git history, which provide with a very reliable source of order on changes.
+
+#### When to update
+
+`CHANGELOG.md` must be updated for the package that is expereincing a change and any other
+package in the repository that re-exports that change in one form or another. In general
+an entry is added according to these rules:
+
+* Every breaking change MUST appear in the `CHANGELOG.md`.
+
+* Every non-breaking change SHOULD appear in the `CHANGELOG.md`
+
+* Every change that does not change the user facing semantics of the code (eg. changes to
+  documentation, depenency bounds, test suites, addition of internal functions,
+  performance improvements, etc.)  COULD be added to the `CHANGELOG.md`, but they are
+  discouraged. Exceptions should be made for changes that could be valueable to the
+  downstream user.
+
+* Every change to a public sub-library of a package must be added to a separate section in
+  the `CHANGELOG.md` and is versioned together with the main library. Eg. `testlib`:
+
+  ```
+  # Version history for `cardano-ledger-core`
+
+  ## 1.20.2.0
+
+  ### `testlib`
+
+  * Add `Arbitrary` instance for ...
+  ...
+  ```
+
+## Release Process
+
+Usually a package release will happen directly from `master` branch. Current release
+process even allows for a release to happen from a commit that is slightly behind `HEAD`
+of `master` branch, but every major release must happen from `master`. On a rare occasion
+when a bugfixes need to be backported to an older version, a release can happen from an
+ephemeral release branch. Such a release branch must follow the same procedure of a PR
+review and CI, that is why it should be prefixed with `release/some-branch-name`, cause
+taht will trigger the CI. should be followed during a bugfix release to an older version,
```

Comment:
```suggestion
that will trigger the CI. should be followed during a bugfix release to an older version,
```
Also seems like there's something missing from the beginning of the sentence.

Reply:
Fixed. Thanks